include/ibamr/IBFEMethod.h

```diff
@@ -765,20 +765,111 @@ class IBFEMethod : public IBStrategy
     int d_max_level_number;
     std::vector<libMesh::EquationSystems*> d_equation_systems;
 
+    /// Number of parts owned by the present object.
     const unsigned int d_num_parts = 1;
+
+    /// Each part has exactly one FEDataManager object.
```

Comment:
Each FEDataManager object manages exactly one part.
```suggestion
    /// Currently, each FEDataManager object is associated with exactly one part.
```